java/ruleXtract/src/main/java/uk/ac/cam/eng/extraction/hadoop/extraction/ExtractorJob.java

```diff
@@ -150,86 +161,44 @@ protected void map(MapWritable key, TextArrayWritable value,
 						ruleInfo.putProvenanceCount(prov2Id.get(prov), ONE);
 					}
 				}
-				ruleInfo.putAlignmentCount(aw, 1);
-				context.write(rw, ruleInfo);
+				ruleInfo.putAlignmentCount(ra.getSecond(), 1);
```

Comment:
No, because One is of type IntWritable and putAlignmentCount takes a second argument of type int, which is a Java primitive.